packages/adblocker-content/adblocker.ts

```diff
@@ -31,6 +31,45 @@ export interface IMessageFromBackground {
   }[];
 }
 
+function debounce(
+  fn: () => void,
+  {
+    waitFor,
+    maxWait,
+  }: {
+    waitFor: number;
+    maxWait: number;
+  },
+) {
+  let delayedTimer: NodeJS.Timeout | -1 = -1;
+  let maxWaitTimer: NodeJS.Timeout | -1 = -1;
```

Comment:
Nit. Why use `-1` as a default instead of `undefined` or `null`?

Reply:
Updated.